internal/integration_test/stdlibs/Makefile

```diff
@@ -1,7 +1,7 @@
 # Note: Some tests might fail if Go has been installed with homebrew because
 # the file system layout is different than what the tests expect.
 # Easiest fix is to install without using brew.
-goroot?=$(shell go env GOROOT)
+goroot?=$(shell go env GOROOT)/src
```

Comment:
`go.mod` is in `/src`. I'm not sure why I seemed to only have huge issues on Windows but it would fail with something like "unable to find go.mod".